editoast/src/views/infra/edition.rs

```diff
@@ -988,9 +987,11 @@ pub mod tests {
         app.fetch(request).assert_status(StatusCode::BAD_REQUEST);
     }
 
-    #[rstest]
+    #[rstest::rstest]
     #[case("TA0", 1000000)]
     #[case("TD1", 15500000)]
+    #[tokio::test(flavor = "multi_thread")]
+    #[ignore]
```

Comment:
I'm having problem with this function that seems to deadlock.

Reply:
With worker_threads = 1 or 2 maybe?

Reply:
Nope, already tried that. At some point of the test, it requires a writable connection to the DB through `.write_owned()`. Since it's behind a `RwLock`, it must wait for all readers to be released and it seems it never happen. I actually tried replacing the `.write_owned()` with `try_write_owned()` which may fail with `TryLockError` and it's exactly what happened.

Reply:
The part I can't explain is why the behavior is different between `async-std` and `tokio` 🤯

Reply:
No idea. Didn't have time to investigate this since, and probably not in the foreseeable future.